ndcube/asdf/resources/schemas/ndcollection-1.0.0.yaml

```
@@ -0,0 +1,30 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "asdf://sunpy.org/ndcube/schemas/ndcollection-1.0.0"
+
+title:
+  Represents the ndcube.ndcollection.NDCollection object
+
+description:
+  Represents the ndcube ndcube.ndcollection.NDCollection object
+
+type: object
+properties:
+  items:
+    type: object
+    patternProperties:
+      ".*":
+        anyOf:
+          - tag: "tag:sunpy.org:ndcube/ndcube/ndcube-1.*"
+          - tag: "tag:sunpy.org:ndcube/ndcube/ndcube_sequence-1.*"
+          # Allow any other objects here as we don't want to restrict what people can save
+          - {}
```

From testing this with the dkist tools I think this needs to be
```suggestion
          - type: object
```
@braingram ?

---

It threw me a fun error about there being no context. I think because the schema evaluated to false.